src/kernel-install/90-uki-copy.install

```diff
@@ -73,6 +73,8 @@ else
     UKI_FILE="$UKI_DIR/$ENTRY_TOKEN-$KERNEL_VERSION.efi"
 fi
 
+ADDON_DIR="$UKI_DIR/$ENTRY_TOKEN-$KERNEL_VERSION.efi.extra.d"
```

Comment:
hmm, this variable's name is a bit of a misnomer, since the ….extra.d/ subdir contains all four kinds of side-cars we have:

* UEFI addons
* credentials
* confext
* sysext

hence rename this variable EXTRA_DIR?

Reply:
also, can you add document the new logic, i.e. in kernel-install(8) there should be a line about these sidecards being copied too where the plugins are discussed under "add"

Reply:
(oh, and maybe also document at the same time that on "remove" (i.e. further down) the 90-uki-copy.install drop-in will not just delete the EFI binary again, but all sidecars too (i.e. the extra.d/ stuff): this it always did, but let's please document that at the same time, to make it symmetric

Reply:
-> #36304.